internal/ports/queue.go

```diff
@@ -0,0 +1,7 @@
+package ports
+
+import "ecommerce_order/internal/domain"
+
+type OrderPublisher interface {
+	Publish(order domain.Order) error
+}

```

Comment:
isso aqui pode ir pra dentro de :

// ports/queue.go

```suggestion
type OrderPublisher interface {
	Publish(order domain.Order) error
}
```

Reply:
dentro de internal mesmo.